clippy.json

```diff
@@ -36,7 +36,10 @@
         "I won't watch...",
         "Do you need recommendations?"
     ],
-    "live.com": "I am homeeee",
+    "live.com": [
+        "I am homeeee",
+        "It looks like you're on Outlook. Would you like help?"
```

Comment:
It looks like you're on Outlook. Would you like **some** help? sounds better to me.

Reply:
I changed it.